.github/workflows/publish-extensions.yml

```diff
@@ -9,20 +9,74 @@ env:
   SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
 
 jobs:
-  pre_run:
+  pre-run:
     runs-on: ubuntu-latest
     steps:
       - name: Cancel Previous Runs
         uses: styfle/cancel-workflow-action@ad6cb1b847ffb509a69b745b6ee2f1d14dfe14b8
         with:
           access_token: ${{ github.token }}
 
+  build-extension:
+    runs-on: ubuntu-latest
+    outputs:
+      new_version: ${{ steps.extract_version.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/provision
+
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.1.1
+
+      - name: Build project
+        env:
+          IS_PUBLISHING: true
+        run: yarn build
+
+      - uses: actions/upload-artifact@v2
+        name: Upload build artifact
+        with:
+          name: hiro-wallet
+          path: dist
+
+  create-github-release:
```

Comment:
As an alternative, the `semantic-release` plugin can create the GH release for you. ([See how to set this up here](https://github.com/hirosystems/gh-actions-example/blob/main/package.json#L55))

Reply:
I remember trying to set this up in the desktop wallet. I found it much easier to manage the configuration directly via the action than via semantic release, I think some of the options are limited